vmngclient/dataclasses.py

```diff
@@ -2,6 +2,7 @@
 from typing import List, Optional
 
 from attr import define, field  # type: ignore
+from pydantic import BaseModel, Field  # type: ignore
```

Comment:
I think this could be uncommented, recently we have added pydantic to additional dependencies for mypy (pre-commit)